server/src/messages/feature/settingsMessages.ts

```diff
@@ -0,0 +1,25 @@
+import { getLocalizedMessage } from '../../services/domain/localization.service';
+
+export const chooseLanguageMessage = (locale: string): string =>
+    getLocalizedMessage(locale, ['Choose language']).join();
+
+export const languageIsNotSupportableMessage = (
+    locale: string | null
+): string =>
+    getLocalizedMessage(locale, [
+        'Language is not supported 😔',
+        '. Choose from the list',
+    ]).join();
+
+export const cannotSetupLanguageMessage = (locale: string): string =>
+    getLocalizedMessage(locale, [
+        `Sorry, I cannot setup the ${locale} for you. I am really sorry  🙇🏽`,
+        `. Probably my database is out of reach. Try latter`,
```

Comment:
`Sorry, I cannot setup the ${locale} for you. I am really sorry  🙇🏽`,
        `. Probably my database is out of reach. Please try a bit latter`

@danbilokha please change message to be

Reply:
which one? :D

Reply:
The one I've posted. It jsut looks more polite when adding please to it